neo4j-apps/graph-examples
# GGP-386: replace old api with graphql

How to test, first run in the terminal:

1. `nvm use` to select the right node version
2. `yarn` to install dependencies
3. `yarn start` to star the server
4. open it in your browser http://localhost:3000/
5. you should be able to navigate and search

Note: play as a browser guide only works if you have neo4j desktop installed
